examples/mongodbatlas_privatelink_endpoint/aws/cluster-geosharded/README.md

```diff
@@ -4,8 +4,8 @@ Setup [regionalized private endpoints](https://www.mongodb.com/docs/atlas/securi
 
 
 ## Gotchas
-- Ensure `mongodbatlas_cluster` depends_on `mongodbatlas_private_endpoint_regional_mode`
-- Despite being properly output, connection strings _may not be applied_ to `mongodbatlas_cluster` resource when changing regional mode enabled. This means the connection_strings may not exist in terraform state until the next `terraform apply`.
+- Ensure `mongodbatlas_advanced_cluster` depends_on `mongodbatlas_private_endpoint_regional_mode`
+- Despite being properly output, connection strings _may not be applied_ to `mongodbatlas_advanced_cluster` resource when changing regional mode enabled. This means the connection_strings may not exist in terraform state until the next `terraform apply`.
```

Comment:
Should the connection_strings be surrounded by single backticks?

Also, in the expression "terraform state" -- should Terraform be capitalized there?

Reply:
you're right, thanks, changed in: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2453/commits/72387e7911dc961b7d12e11ceb5cbfe04d84bfc6